sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -24,8 +21,7 @@ class AnonymousACRExchangeClient(object): # pylint: disable=client-accepts-api-v
     :type credential: ~azure.core.credentials.TokenCredential
     """
 
-    def __init__(self, endpoint, **kwargs):  # pylint: disable=missing-client-constructor-parameter-credential
-        # type: (str, Dict[str, Any]) -> None
+    def __init__(self, endpoint: str, **kwargs: Dict[str, Any]) -> None:  # pylint: disable=missing-client-constructor-parameter-credential
```

Comment:
We don't need to add the type hint for kwargs

Reply:
I have removed all type hints for "kwargs" in convenience layer.